lib/log.js

```diff
@@ -0,0 +1,49 @@
+const bunyan = require('bunyan');
+
+const { name, version } = require('../package');
+const config = require('./config');
+
+const options = {
+  name,
+  level: config.LOG_DISABLE ? (bunyan.FATAL + 1) : config.LOG_LEVEL, // turn off logging https://github.com/trentm/node-bunyan/pull/148#issuecomment-63018668
+  version,
+  src: true,
+  obfuscate: [
+    'body.password'
+  ],
+  period: '1d', // daily rotation
+  count: 3, // keep 3 back copies
+  obfuscatePlaceholder: config.LOG_OBFUSCATE_PLACEHOLDER,
+  streams: [],
+  genReqId: (req) => { return req.id; }
```

Comment:
`genReqId: req => req.id` Also works :). Or even `genReqId: ({id}) => id`, or `genReqId({ id }) { return id }`. But of course, whatever :)

Reply:
nice!